examples/auto_ml_checkpoint.py

```diff
@@ -132,3 +131,6 @@ def mean_squared_error(actual_outputs, expected_outputs):
     assert isinstance(outputs, np.ndarray)
 
     pipeline.flush_all_cache()
+
+if __name__ == "__main__":
+    main(0.1)
```

Comment:
Note: I believe this will be executed as a main when compiling the documentation. We might need to make this number much lower, because 200 iterations is a lot of iterations.